SpatialGDK/Source/SpatialGDK/Public/SpatialView/CommandMessages.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+#include "Misc/Optional.h"
+#include "Containers/UnrealString.h"
+#include <improbable/c_worker.h>
+
+namespace SpatialGDK
+{
+
+struct CreateEntityRequest
+{
+	Worker_RequestId RequestId;
+	// todo this should be a owning entity state type.
```

Comment:
Tracking ticket?

Reply:
See plenty of TODOs actually, generally we create tracking tickets, but not hard requirement.

Reply:
In this case, since it's in active development, I don't mind if there aren't tracking tickets. It would be nice to have them, but it will likely be done as a drive-by in future work, and we're not good at tracking those in tickets.